java/src/main/java/lesson03/README.md

```diff
@@ -125,6 +125,19 @@ Notice that we also add a couple additional tags to the span with some metadata
 and we mark the span with a `span.kind=client` tag, as recommended by the OpenTracing
 [Semantic Conventions][semantic-conventions]. There are other tags we could add.
 
+#### Handling Errors
+Since we turned our single-binary program into a distributed application that makes remote calls, we need to handle errors that may occur during communications. It is a good practice to tag the span with the tag `error=true` if the operation represented by the span failed. So, let's go ahead and update the `catch` section of `get_http` function with below code snippet:
+
+```java
+catch (IOException e) {
+    Tags.ERROR.set(tracer.activeSpan(), true);
+    tracer.activeSpan().log(ImmutableMap.of("event", "error", "value", e));
```

Comment:
the semantic convention recommends `error.object`, not `value`, e.g. https://github.com/opentracing/opentracing-go/blob/0dfafacef64a117a0eeb0fb25c8e3a6ee9ff4efa/log/field.go#L125

Reply:
Noted. Will use `tracer.activeSpan().log(ImmutableMap.of(Fields.EVENT, "error", Fields.ERROR_OBJECT, response));` code.